yowsup/demos/jsonpost/layer.py

```diff
@@ -0,0 +1,71 @@
+from yowsup.layers.interface import YowInterfaceLayer, ProtocolEntityCallback
+from yowsup.layers.network   import YowNetworkLayer
+
+import datetime, sys, json, time, requests, os
+
+class EchoLayer(YowInterfaceLayer):
+
+    def onEvent(self, layerEvent):
+        print layerEvent.getName()
```

Comment:
what about python 3+ ? In any way logger should be used instead of print statements